src/js/msp/MSPHelper.js

```diff
@@ -1687,6 +1691,10 @@ MspHelper.prototype.crunch = function(code) {
                             .push8(RX_CONFIG.rcSmoothingDerivativeCutoff)
                             .push8(RX_CONFIG.rcSmoothingInputType)
                             .push8(RX_CONFIG.rcSmoothingDerivativeType);
+                        if (semver.gte(CONFIG.apiVersion, "1.42.0")) {
+                            buffer.push8(RX_CONFIG.usbCdcHidType)
+                                .push8(RX_CONFIG.rcSmoothingAutoSmoothness);
```

Comment:
Usually in this file we align the `.` under each push 👾

Reply:
Fixed